src/lightmap.cpp

```diff
@@ -653,8 +653,8 @@ lit_level map::apparent_light_at( const tripoint &p, const visibility_variables
     if( cache.u_clairvoyance > 0 && dist <= cache.u_clairvoyance ) {
         return lit_level::BRIGHT;
     }
-    const field_type_str_id fd_clairvoyant( "fd_clairvoyant" );
-    if( fd_clairvoyant.is_valid() && field_at( p ).find_field( fd_clairvoyant ) ) {
+    static const field_type_str_id fd_clairvoyant( "fd_clairvoyant" );
```

Comment:
This should probably be a global static so that this function doesn't need to do an atomic operation every time it runs.